src/main/java/utils/SwipeHelper.java

```diff
@@ -2,18 +2,73 @@
 
 import io.appium.java_client.MobileBy;
 import io.appium.java_client.MobileElement;
+import io.appium.java_client.PerformsTouchActions;
+import io.appium.java_client.TouchAction;
+import io.appium.java_client.touch.WaitOptions;
+import io.appium.java_client.touch.offset.PointOption;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.WebElement;
+
+import java.time.Duration;
 
 public class SwipeHelper {
 
-    public static MobileElement androidHorizonalSwipe(String carousel, String option){
+    public static MobileElement androidHorizonalSwipe(String carousel, String option) {
         return BrowserManagement.getDriver().findElement(MobileBy.AndroidUIAutomator(
-                "new UiScrollable("+carousel+").setAsHorizontalList()." +
-                "scrollIntoView("+option+")"));
+                "new UiScrollable(" + carousel + ").setAsHorizontalList()." +
+                        "scrollIntoView(" + option + ")"));
     }
 
-    public static MobileElement androidVerticalSwipe(String carousel, String option){
+    public static MobileElement androidVerticalSwipe(String carousel, String option) {
         return BrowserManagement.getDriver().findElement(MobileBy.AndroidUIAutomator(
-                "new UiScrollable("+carousel+").setAsVerticalList()." +
-                        "scrollIntoView("+option+")"));
+                "new UiScrollable(" + carousel + ").setAsVerticalList()." +
+                        "scrollIntoView(" + option + ")"));
+    }
+
+    /**
+     * Método de https://github.com/sunilpatro1985/AppiumTest_Java_And_iOS/blob/master/src/main/java/base/Util.java
+     */
+    public static void scrollDown() {
+        Dimension dimension = BrowserManagement.getDriver().manage().window().getSize();
+        int scrollStart = (int) (dimension.getHeight() * 0.5);
+        int scrollEnd = (int) (dimension.getHeight() * 0.2);
```

Comment:
Was this tested in different resolutions?

Reply:
In a 5 inch device and 5.6 inch device